packages/affiliates/gas-rebate/VoterGasRebate.js

```diff
@@ -35,6 +35,16 @@ const argv = require("minimist")(process.argv.slice(), {
 const fs = require("fs");
 const path = require("path");
 
+// Locally stored list of DesignatedVoting wallets to substitute with their hot wallets:
+let addressesToReplace = {};
```

Comment:
What is the layout of this mapping? address => ?

Reply:
Its just address => bool. If I make it an array, then exporting it is a bit uglier (require("addressesToReplace).addressArray). I'm not strongly opinionated. WDYT is the best format?

Reply:
I think an array would be slightly more intuitive. Can the top level in the JSON file be an array?

[
  "0x1234",
  "0x1234"
]

Reply:
Oh wow didn't realize you could export an array in a json file. Done